packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx

```diff
@@ -278,6 +277,41 @@ const WorkspaceHeader: React.FC<{}> = () => {
           }
         />
         <hr className={`${CLASS_NAME}__vertical_border`} />
+        <div className={`${CLASS_NAME}__help_popover`}>
+          <SelectMenu
+            title={"Help"}
+            buttonStyle={EnumButtonStyle.Text}
+            icon="chevron_down"
+            openIcon="chevron_up"
+            className={`${CLASS_NAME}__help_popover__menu`}
+          >
+            <SelectMenuModal align="right">
+              <SelectMenuList>
+                {HELP_MENU_LIST.map((route: HelpMenuItem, index) => (
+                  <SelectMenuItem
+                    closeAfterSelectionChange
+                    onSelectionChange={() => {
+                      !route.url && handleContactUsClick();
+                    }}
```

Comment:
This solution is OK only for the current situation where we have a single item with a callback, but we need to implement a more generic solution.

we can use `itemData` on SelectMenuItem to pass a name of a command and then in a callback function trigger a function based on the command name

{ name: "Contact Us", url: null, itemData: COMMAND_CONTACT_US },